### ISSUE TITLE
Default strategies not available when supplying custom strategies

### ISSUE THREAD

**Issue by AroglDarthu** [stiano/unleash-client-dotnet#13]
https://github.com/stiano/unleash-client-dotnet/blob/6a437ed9c4c5633490b2ff7893148b07108ffd0e/src/Unleash/DefaultUnleash.cs#L99

I would have expected the custom strategies to be added to the default ones. Unfortunately the default ones are skipped.

If this is a deliberate design decision, please consider exposing a public property to access the default strategies, so one can choose to pass these in as part of the custom strategies.

**Comment by stiano** [comment]
@AroglDarthu

Yeah, the api could have been clearer on that part. I will expose a public property for default strategies for the next release.

**Comment by ivarconr** [comment]
Adressed in https://github.com/Unleash/unleash-client-core
